File: CS/Converters/FontStyleConverter.cs
using System;
using System.Windows;
using System.Windows.Data;
namespace DXRibbonNotepad.Converters {
    public class FontStyleConverter : IValueConverter {
        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture) {
            if((bool)value)
                return FontStyles.Italic;
            return FontStyles.Normal;
        }
        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture) {
            throw new NotImplementedException();
        }
    }
}

File: VB/Converters/FontStyleConverter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows
Imports System.Windows.Data
Namespace DXRibbonNotepad.Converters
	Public Class FontStyleConverter
		Implements IValueConverter
		Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.Convert
			If CBool(value) Then
				Return FontStyles.Italic
			End If
			Return FontStyles.Normal
		End Function
		Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.ConvertBack
			Throw New NotImplementedException()
		End Function
	End Class
End Namespace